core_data_modules/analysis/sample_messages.py

```diff
@@ -0,0 +1,92 @@
+import random
+import sys
+from collections import OrderedDict
+
+from core_data_modules.analysis import analysis_utils
+
+sample_messages_keys = ["Episode", "Code Scheme", "Code", "Sample Message"]
```

Comment:
Wondering whether we should consider a different name other than sample to cover the other use cases too.

Reply:
Good point, I've renamed the "Sample Message" header -> "Message".